### PR TITLE
feat(page): all page

### PR DESCRIPTION
**Pull request by bestenevoy**

### CODE REVIEW

File: src/components/preview.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+export default function ThemePreview ({manifest, gistsid, className = ''}: IPreviewProps) {
+    const [src, setSrc] = useState(`https://${AWSHOST}/desktop/czg.vercel.app/theme/${gistsid}.jpg`)
+    let errored = true
+    const onError = () => {
+        if (errored) {
+            setSrc(`_next/image?url=/api/snapshot/${gistsid}&w=1920&q=75`)
+            errored = true
+        }
```
Comment (somarlyonks) [inline src/components/preview.tsx:20]: What does `errored` do here?

Comment (bestenevoy) [reply src/components/preview.tsx]: prevent again catch

File: src/config.ts [hunk 2]
```diff
@@ -25,3 +25,9 @@ export const FETCH_GIST_CACHE_LIFETIME = 60 // seconds
 export const FETCH_GISTS_CACHE_LIFETIME = 60 // seconds
 
 export const SUBMIT_CHANNEL = 'submit-channel'
+
+export const AWSHOST = 'csszen-screenshot.s3.us-east-2.amazonaws.com'
+
+export const SNAPSHOT_SERVER = 'https://screenshot-omega.vercel.app'
+
+export const COUNT_PER_PAGE = 6
```
[the remark this answers is not in the document]
Comment (bestenevoy) [reply src/config.ts]: good thinking

**Review by somarlyonks — APPROVED** [review]

**Review by somarlyonks — APPROVED** [review]
squash

**Pull request merged**